scale/docs/rest/v6/job_type.rst

```diff
@@ -840,579 +792,80 @@ Response: 200 OK
-| .. code-block:: javascript                                                                                              |
-|                                                                                                                         |
-|    {                                                                                                                    |
-|        "warnings": [                                                                                                    |
-|            "id": "settings",                                                                                            |
-|            "details": "Missing configuration for interface required setting"                                            |
-|        ]                                                                                                                |
-|    }                                                                                                                    |
-+-------------------------------------------------------------------------------------------------------------------------+
+| is_valid           | Boolean           | Indicates if the given fields were valid for creating a new job type. If this  |
+|                    |                   | is true, then submitting the same fields to the /job-types/ API will           |
+|                    |                   | successfully create a new job type.                                            |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| errors             | Array             | Lists any errors causing *is_valid* to be false. The errors are JSON objects   |
+|                    |                   | with *name* and *description* string fields.                                   |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| warnings           | Array             | Lists any warnings found. Warnings are useful to present to the user, but do   |
+|                    |                   | not cause *is_valid* to be false. The warnings are JSON objects with *name*    |
+|                    |                   | and *description* string fields.                                               |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+
+.. _rest_v6_job_type_edit:
+
+v6 Edit Job Type
+----------------
+
+**Example PATCH /v6/job-types/{name}/{version}/ API call**
+
+Request: PATCH http://.../v6/job-types/test/1.0.0/
+
+ .. code-block:: javascript  
+    {
+      "icon_code": "012F",
+      "is_active": true,
+      "is_paused": false,
+      "max_scheduled": 1,
+      "configuration": {
+        "version": "2.0",
```

Comment:
Same comment as above about removing `version` from the configuration JSON. This should be done everywhere in the v6 Job Type docs.